test/Analysis/xtu-main_usr.cpp

```diff
@@ -76,4 +85,9 @@ int main() {
 
   clang_analyzer_eval(fun_using_type_alias_template() == 3); // expected-warning{{TRUE}}
   clang_analyzer_eval(fun_using_pack_expansion() == 4); // expected-warning{{TRUE}}
+
+  // Note: the analysis of template function is not supported yet.
+  Ptr p;
+  clang_analyzer_eval(p.a<void>() == 1);       // expected-warning{{UNKNOWN}}
```

Comment:
This is surprising. Why is it not supported? Where is the limitation?

Reply:
In the meantime, we discovered the reason, and code snippets like this should not happen in the checked codebase.